tests/test_models.py

```diff
@@ -209,7 +209,9 @@ def _check_constraints(self):
 
         Entity embeddings have to have at most unit L2 norm.
         """
-        assert all_in_bounds(self.instance.entity_embeddings(indices=None).norm(p=2, dim=-1), high=1.0, a_tol=EPSILON)
+        assert all_in_bounds(
+            self.instance.entity_representations[0](indices=None).norm(p=2, dim=-1), high=1.0, a_tol=EPSILON
```

Comment:
same question as the other PR, is this change necessary?

Reply:
linking the other issue [here](https://github.com/pykeen/pykeen/pull/955#discussion_r887738228); discussion over there